.pylintrc

```diff
@@ -66,7 +66,7 @@ ignore-imports=yes
 #    too-many-statements
 # and fix those issues.
 
-disable=duplicate-code
+disable=duplicate-code, no-member
```

Comment:
Why add this? Try not to.

Reply:
The problem is with [protobuf generated class objects](https://github.com/PyCQA/pylint/issues/1864). These classes are generated in a way that no explicit member is defined on them (you can use anything for attribute name). Thus anytime we access a pylint object's attributes, it yields the error of `no-member`. Pylint devs have denied adding any feature for this and so either we disable it at every line in the code wherever we access it or globally. Since this is disabled globally in Oppia, I decided to do so here, too.

Reply:
Added a comment,

Reply:
Thanks -- but please add the comment *above* the line in question, not below it.

Reply:
Done